lib/forms.js

```diff
@@ -15,6 +15,7 @@ exports.widgets = require('./widgets');
 exports.fields = require('./fields');
 exports.render = require('./render');
 exports.validators = require('./validators');
+exports.tag = require('./tag');
```

Comment:
This isn't necessary - `require('forms/lib/tag')` works now. Please revert this.